scripts/yrd/install

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+# get previous location - should be 
+START_DIR="$(pwd)"
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+mkdir "$BASEDIR/tmp"
+cd "$BASEDIR/tmp"
+git clone https://github.com/kpcyrd/yrd.git
+cd yrd
+# install dependencies
+sudo apt-get install python3 python3-pip python3-argh python3-requests
+# install yrd
+sudo -H python3 ./setup.py -v install
+# install and enable the service
+cd "$START_DIR"
+sudo cp yrd.service /etc/systemd/system/
```

Comment:
Don't need to change work directory.   And relaying on work directory is not the correct action as work directory can be different then where the file is located.

Instead use `BASE_DIR`
`sudo cp "$BASE_DIR/yrd.service" "/etc/systemd/system/yrd.service"`